convene-web/app/views/spaces/edit.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with model: @space do |form| %>
+<%= form.text_field :name %>
+<% end %>
+
+<%- @space.members.each do |member| %>
+  <%= member.name %>
+  <%= member.email %>
+<%- end %>
+
+<%= form_with model: @space.space_memberships.new do |member_form| %>
+
+<% end %>
```

Comment:
Before we merge, we will probably want to change this to use the new `@space.invitations` relationship instead of the `@space.space_memberships` relationship.